api-docs-slate/source/includes/_clients.md

```diff
@@ -17,94 +17,99 @@ testnet   | 18332
 --------- | -----------                  | -----------
-prefix    | dir path                     | This accepts directory where DBs and `bcoin.conf` are located.
-network   | `main`, `testnet`, `regtest` | This will configure which network to load, also where to look for config file
+network   | `main`, `testnet`, `regtest` | This will configure which network to load, also where to look for `bcoin.conf` file
 uri, url  | Base HTTP URI                | This can be used for custom port
-api-key   | secret                       | Secret used by RPC for auth.
+api-key   | _string_                       | Secret used by RPC for authorization
 
 ### Wallet Specific
 
 Config    | Options         | Description
 --------- | -----------     | -----------
-id        | primary, custom | specify which account to use by default
-token     | token str       | Token specific wallet
+id        | _string_ | specify which account to use by default
+token     | _string_       | Token specific wallet
+
+
+```shell--visible
+# Example bcoin.conf syntax:
+network: main
+prefix: ~/.bcoin
+api-key: bikeshed
+```
+
+### bcoin.conf file
+
+[A sample bcoin.conf file is included in the code repository](https://github.com/bcoin-org/bcoin/blob/master/etc/sample.conf)
+
+
+
 
 <aside class="notice">
 Some commands might accept additional parameters.
 </aside>
 
-## Using Javascript Client
+## Using Javascript Clients
 
 ```javascript--visible
-const bcoin = require('bcoin');
-const Client = bcoin.http.Client;
-const Wallet = bcoin.http.Wallet;
-
-const client = new Client({
-  network: 'testnet',
-  uri: 'http://localhost:18332'
-});
-
-const wallet = new Wallet({
-  network: 'testnet',
-  uri: 'http://localhost:18332',
-  id: 'primary'
-});
+const {NodeClient, WalletClient} = require('bclient');
+const {Network} = require('bcoin');
+const network = Network.get('regtest');
+
+const clientOptions = {
+  network: network.type,
+  port: network.rpcPort,
+  apiKey: 'api-key'
+}
+
+const walletOptions = {
+  network: network.type,
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const client = new NodeClient(clientOptions);
+const wallet = new WalletClient(walletOptions);
 ```
 
-You can also use api with Javascript Library (used by `bcoin-cli`).
-There are two objects: `bcoin.http.Client` for general API and `bcoin.http.Wallet` for wallet API.
+You can also use the API with a Javascript library (used by `bcoin-cli`).
+There are two objects: `NodeClient` for general API and `WalletClient` for wallet API.
+`bcoin` also provides an object `Network` and its method `get` which will return the appropriate configuration paramaters for a specified network.
 
-`bcoin.http.Client` options:
+`NodeClient` and `WalletClient` options:
 
 Config    | Type                         | Description
 --------- | -----------                  | -----------
 network   | `main`, `testnet`, `regtest` | Network to use (doesn't lookup configs by itself)
```

Comment:
Type should still be string as this is indicating primitive type that it's expecting. Then in the description you can something like "should be one of `main`, `testnet`, `regtest`, or `simnet`" (also I always forget but is it `main` or `mainnet`?)

Reply:
main: https://github.com/bcoin-org/bcoin/blob/master/lib/protocol/networks.js#L25